source/DSCResources/DSC_PostgreSqlDatabase/DSC_PostgreSqlDatabase.psm1

```diff
@@ -125,14 +128,14 @@ function Set-TargetResource
             Write-Verbose -Message ($script:localizedData.CreatingDatabase -f $DatabaseName)
             Invoke-Command -ScriptBlock {
```

Comment:
Is there any benefit to using Start-Process here?  When calling an .exe from PowerShell I like to use Start-Process -Wait so ensure the script will not progress until the .exe is finished.  I have found in the past that PowerShell will call the .exe and then keep on going.  This would also allow you to inspect the exitCode to verify success.  It might also remove the requirement to change the built-in variables.

Reply:
We can try it and see how it goes.